pkg/server/server_test.go

```diff
@@ -143,10 +148,22 @@ func TestHealthzHandler(t *testing.T) {
 func TestCancelHandler(t *testing.T) {
 	s := initServer()
 
-	testBody := strings.NewReader("{\"Filter\":{\"id\":1}}")
+	cID, err := strconv.ParseFloat("10", 32)
+	if err != nil {
+		t.Fatal(err)
+	}
```

Comment:
We should just pass a uint in `q` instead of this strconv/float dance.

Reply:
yeah, my bad - left something in from testing I didn't need to keep